pkg/releasetesting/environment.go

```diff
@@ -96,7 +96,7 @@ func (env *Environment) streamError(info string) error {
 }
 
 func (env *Environment) streamFailed(name string) error {
-	msg := fmt.Sprintf("FAILED: %s, run `kubectl logs %s --namespace %s` for more info", name, name, env.Namespace)
```

Comment:
now that I've run the comparison of current behavior vs this change, I wonder if this line should remain as is? 
* it's still helpful when `--cleanup` is not used
* people may be expecting it as is, and using it this way
* it might not be so useful, but it doesn't impact anything to leave it as is

???

Reply:
I'd keep it as-is if it's irrelevant to the changes involved in this PR.